scripts/xsslint/tests/test_linters.py

```diff
@@ -421,9 +421,17 @@ def test_javascript_interpolate(self, data):
 
     @data(
         {'template': '_.escape(message)', 'rule': None},
-        {'template': 'anything.escape(message)', 'rule': JAVASCRIPT_LINTER_RULESET.javascript_escape},
+        {'template': 'anything.escape(message)', 'rule': None},
+        {'template': 'anythingescape(message)', 'rule': None},
+        {'template': '$escape(message)', 'rule': None},
+        {'template': '_escape(message)', 'rule': None},
+        {'template': 'escape(message)', 'rule': JAVASCRIPT_LINTER_RULESET.javascript_escape},
+        {'template': '(escape(message))', 'rule': JAVASCRIPT_LINTER_RULESET.javascript_escape},
+        {'template': '  escape(message))', 'rule': JAVASCRIPT_LINTER_RULESET.javascript_escape},
+        {'template': 'window.escape(message)', 'rule': JAVASCRIPT_LINTER_RULESET.javascript_escape},
+        {'template': '(window.escape(message)', 'rule': JAVASCRIPT_LINTER_RULESET.javascript_escape},
```

Comment:
Could we have some or all of these get a brief trailing comment explaining why?  Or maybe one comment before the success and one before the failures?  What is `window.escape`?

Reply:
Update: Maybe update the linting method docstring to clarify this _dangerous_ escape methods.  Looks like `window.escape` is deprecated?  Also, consider updating https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/preventing_xss/preventing_xss.html#javascript-escape

Reply:
That linked section in the docs appears to still be relevant and correct. The changes here don't invalidate that. :)

Reply:
Yes. `escape` is in the global namespace, and `window` is the global namespace for browsers, so `escape == window.escape`.